elpy.el

```diff
@@ -444,6 +433,36 @@ option is `pdb'."
 
     (define-key map (kbd "M-TAB") 'elpy-company-backend)
 
+    ;; key bindings for elpy-shell
+    (define-key map (kbd "C-c e e") 'elpy-shell-send-statement)
```

Comment:
`C-c <char>` is a reserved key for the user and should not be used by modes. We could bind something to `C-u C-RET` or `C-u C-c C-c` for this, maybe. Or you could use `C-c C-y` as a prefix if you like (sending it to the other buffer is a bit like yanking it there maybe).

Thinking about stealing `C-c RET` back from importmagic, too …